packages/itwinui-react/src/core/ProgressIndicators/ProgressRadial.tsx

```diff
@@ -68,6 +69,7 @@ export const ProgressRadial = React.forwardRef((props, forwardedRef) => {
   const statusMap = {
     negative: <SvgImportantSmall aria-hidden />,
     positive: <SvgCheckmarkSmall aria-hidden />,
+    warning: <SvgImportantSmall aria-hidden />,
```

Comment:
Is this icon correct?

cc @FlyersPh9

Reply:
[That is the icon used in the error radial progress indicator](https://github.com/iTwin/iTwinUI/blob/f29c68369b01c92341825bfdf136c99a5f109505/apps/css-workshop/pages/progress-indicator.html#L24C3-L24C62) & what I would use for the warning variant as well. 👍
We didn't use `status-error` or `status-success` in the radial progress indicators because it would create a circle icon within the circular progress indicator, so for now `important-small` seems to be the best option that we currently have.